impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -388,6 +414,11 @@ std::shared_ptr<Texture> AllocatorVK::OnCreateTexture(
   return std::make_shared<TextureVK>(context_, std::move(source));
 }
 
+void AllocatorVK::IncrementFrame() {
+  frame_count_++;
+  raster_thread_id_ = std::this_thread::get_id();
```

Comment:
This is an assumption that the thread which invokves IncrementFrame is the raster thread and will do most of the allocation. This is important to keep worker thread allocations in the default pool.